packages/wdio-browserstack-service/src/types.ts

```diff
@@ -324,7 +324,7 @@ export interface FeaturesUsageData {
 
 export interface CBTData {
     uuid: string
-    integrations: IntegrationObject
+    integrations: { [index: string]: IntegrationObject }
```

Comment:
Why is this change needed?

Reply:
The existing type doesn't include hub provided field. 
This is also done for the `TestData` type in the same file. Implemented the same